modules/support/src/lib.rs

```diff
@@ -436,3 +472,45 @@ impl<AccountId, Balance: Default + Copy, NegativeImbalance: Imbalance<Balance>>
 pub trait Contains<T> {
 	fn contains(t: &T) -> bool;
 }
+
+/// A mapping between `AccountId` and `EvmAddress`.
+pub trait AddressMapping<AccountId> {
+	fn get_account_id(evm: &EvmAddress) -> AccountId;
+	fn get_evm_address(account_id: &AccountId) -> Option<EvmAddress>;
+	fn get_or_create_evm_address(account_id: &AccountId) -> EvmAddress;
+	fn get_default_evm_address(account_id: &AccountId) -> EvmAddress;
```

Comment:
Forgot to add review comments here. I think the naming here is not consist with other parts, we don't usually `get_` prefix in rustlang functions.